server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1939,14 +1939,14 @@ public File exportCertificates(
 
         List<CertificateSerialDto> allCerts = new LinkedList<>();
         for (Long id : entCertService.listEntitlementSerialIds(consumer)) {
-            allCerts.add(new CertificateSerialDto(id));
+            allCerts.add(new CertificateSerialDto(id.toString()));
```

Comment:
I can't comment on the right line here, but this function appears to be using the wrong DTO. We should be using the DTOs from the API package, not the older packages. Can probably save some work and confusion by updating this method to use the right objects.